django_freeradius/api/views.py

```diff
@@ -43,6 +45,26 @@ def post(self, request, *args, **kwargs):
 postauth = PostAuthView.as_view()
 
 
+class AccountingFilter(filters.FilterSet):
+    start_time = filters.IsoDateTimeFilter(name='start_time')
+    stop_time = filters.IsoDateTimeFilter(name='stop_time')
+    open = filters.BooleanFilter(label='Open', method='filter_open')
```

Comment:
let's change this to `is_open` to avoid overriding the `open` reserved keyword, only noticed this now.

Remove the `label` if you can for consistency, if we add it we should add it on all the other fields

Reply:
Yeah I was going to tell you that.